src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall(){
+        int number = new Random().nextInt(100);
+        return new ColorSupplier().getRandomColor() + number;

```

Comment:
Random and ColorSupplier must be class fields
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)